test/component/home/static_home_test.dart

```diff
@@ -84,7 +86,40 @@ void main() {
     expect(homePage, findsOneWidget);
 
     //Check empty post message is displayed
-    final emptyPostMessage = find.byKey(HomeFeed.emptyHomeFeedKey);
+    final emptyPostMessage = find.byKey(PostFeed.emptyfeedKey);
     expect(emptyPostMessage, findsOneWidget);
+
+    //Check the new post button text is displayed
+    final newPostButtonText = find.byKey(PostFeed.newPostButtonTextKey);
+    expect(newPostButtonText, findsOneWidget);
+  });
+
+  testWidgets(
+      "check correct number of navigation bottom bar elements are displayed",
+      (tester) async {
+    final homePageWidget = ProviderScope(
+      overrides: [postList.overrideWithValue(List.empty())],
+      child: const MaterialApp(
+        title: "Proxima",
+        home: HomePage(),
+      ),
+    );
```

Comment:
There is a lot of code repetition in the tests, this is an example.
I don't think this is really important to fix right now, because it would require extensive refactoring, without huge interest. This time is probably best invested somewhere else for now. However, I think that keeping this in mind for the following PRs might be interesting. :)

Reply:
Good catch